napari/layers/labels/_labels_key_bindings.py

```diff
@@ -73,11 +73,11 @@ def new_label(layer: Labels):
 
 
 @register_label_action(
-    trans._("Set the currently selected label to the background."),
+    trans._("Set the selected label to the background label or vice versa."),
```

Comment:
```suggestion
    trans._("Swap between the selected label and the background label."),
```
How 'bout something like this? I like swap or switch as verbs instead of set seems better to me.

Reply:
Agreed, your description looks better, updated the docstrings.